packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx

```diff
@@ -35,31 +55,28 @@ const PlatformDropdown = ({ app_routing_history, closeDrawer, platform_config })
             closeDrawer();
         }
     };
+    const { is_pre_appstore } = React.useContext(PlatformContext);
 
     useOnClickOutside(ref, handleClickOutside, () => isDesktop());
 
     const platform_dropdown = (
         <div className='platform-dropdown'>
             <Div100vhContainer className='platform-dropdown__list' height_offset='156px' is_disabled={isDesktop()}>
-                {platform_config.map((platform, idx) => (
-                    <div key={idx} onClick={closeDrawer} ref={ref}>
-                        {platform.link_to ? (
-                            <BinaryLink
-                                to={platform.link_to}
-                                // This is here because in routes-config it needs to have children, but not in menu
-                                exact={platform.link_to === routes.trade}
-                                className='platform-dropdown__list-platform'
-                                isActive={() => getActivePlatform(app_routing_history) === platform.name}
-                            >
-                                <PlatformBox platform={platform} />
-                            </BinaryLink>
-                        ) : (
-                            <a href={platform.href} className='platform-dropdown__list-platform'>
-                                <PlatformBox platform={platform} />
-                            </a>
-                        )}
-                    </div>
-                ))}
+                {platform_config.map((platform, idx) => {
+                    const should_hide_dropdown_item =
+                        (platform.name === getPlatformSettings('mt5').name ||
+                            platform.name === getPlatformSettings('dxtrade').name) &&
+                        is_pre_appstore;
+                    return (
+                        <div key={idx} onClick={closeDrawer} ref={ref}>
```

Comment:
Don't use index as `key` 
https://articles.wesionary.team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9